### PR TITLE
docs(store): document selector testing using .projector

### PR DESCRIPTION
**Pull request by 03byron**
This PR documents #290.

### CODE REVIEW

File: docs/store/testing.md [hunk 1]
```diff
@@ -108,3 +108,40 @@ describe('My Component', () => {
   });
 });
 ```
+### Testing selectors
+If you have selectors composed of many other selectors it may be useful to test them independent from the selectors they are composed of. Therefor selectors contain a reference to the projector function as `.projector`.
```
Comment (brandonroberts) [inline docs/store/testing.md]: Will you change this to "You can use the projector function used by the selector by accessing the `.projector` property`

**Comment by brandonroberts** [comment]
@03byron thanks for the PR! One minor change and this is ready to land.

**Comment by 03byron** [comment]
Thanks for your feedback, @brandonroberts. Just changed it.

**Comment by brandonroberts** [comment]
Thanks!

**Comment by sten82** [comment]
Thanks, that helped me a lot!

**Pull request merged**